.rubocop_rspec_base.yml

```diff
@@ -69,10 +69,6 @@ LineLength:
 MethodLength:
   Max: 15
 
-# Who cares what we call the argument for binary operator methods?
```

Comment:
Are these just in line with 4-0?

Reply:
No, those rules have diverged in https://github.com/rspec/rspec-rails/pull/2272, and we got rid of `.rubocop_rspec_base.yml` altogether when separated `rspec-rails` from the rest of RSpec.